hoodie-client/src/main/java/com/uber/hoodie/io/HoodieCommitArchiveLog.java

```diff
@@ -58,44 +71,84 @@ public HoodieCommitArchiveLog(HoodieWriteConfig config, FileSystem fs) {
             .getArchiveLogPath(config.getBasePath() + "/" + HoodieTableMetaClient.METAFOLDER_NAME);
     }
 
+    private HoodieLogFormat.Writer openWriter() {
+        try {
+            if(this.writer == null) {
+                return HoodieLogFormat.newWriterBuilder()
+                        .onParentPath(archiveFilePath.getParent())
+                        .withFileId(archiveFilePath.getName())
+                        .withFileExtension(HoodieLogFile.DELTA_EXTENSION)
```

Comment:
Delta is specific to the redo log ( that is the delta to be applied on the columnar file ). Create a new extension for the archived fle.